provider/resources.go

```diff
@@ -95,49 +100,14 @@ func Provider() tfbridge.ProviderInfo {
 				Default: &tfbridge.DefaultInfo{
 					EnvVars: []string{"NEW_RELIC_ACCOUNT_ID"},
 				},
-			},
-			"api_key": {
-				Default: &tfbridge.DefaultInfo{
-					EnvVars: []string{"NEW_RELIC_API_KEY"},
-				},
-			},
-			"admin_api_key": {
-				Default: &tfbridge.DefaultInfo{
-					EnvVars: []string{"NEW_RELIC_ADMIN_API_KEY"},
-				},
+				ForceNew: boolRef(true),
```

Comment:
I thought we weren't going to mark anything `ForceNew` to start?

Reply:
My mistake!!! Removing now :)